block-producer/src/block_builder/prover/mod.rs

```diff
@@ -54,12 +53,12 @@ proc.compute_account_root
         movdn.9
         # => [ROOT_i, NEW_ACCOUNT_HASH_i, account_id_i, counter, ...]
 
-        # Move root down (for smt64.set)
-        movdn.8 movdn.8 movdn.8 movdn.8
-        # => [NEW_ACCOUNT_HASH_i, account_id_i, ROOT_i, counter, ...]
+        # Prepare stack for `mtree_set`
+        movup.8 push.64
+        # => [64, account_id_i, ROOT_i, NEW_ACCOUNT_HASH_i, counter, ...]
```

Comment:
Instead of hard-coding 64 here, let's create a constant - e.g., `ACCOUNT_TREE_DEPTH`.